spooq/transformer/mapper.py

```diff
@@ -126,29 +123,31 @@ class Mapper(Transformer):
     def __init__(
         self,
         mapping,
-        ignore_missing_columns=False,
         ignore_ambiguous_columns=False,
-        nullify_missing_columns=False,
-        skip_missing_columns=False,
+        missing_column_handling="raise_error",
         mode="replace",
+        **kwargs
     ):
         super(Mapper, self).__init__()
-        self.logger.warn("Parameter `ignore_missing_columns` is deprecated, use `nullify_missing_columns` instead!")
-        warnings.warn(
-            message="Parameter `ignore_missing_columns` is deprecated, use `nullify_missing_columns` instead!",
-            category=FutureWarning
-        )
         self.mapping = mapping
-        self.nullify_missing_columns = nullify_missing_columns or ignore_missing_columns
-        self.skip_missing_columns = skip_missing_columns
-        if self.nullify_missing_columns and self.skip_missing_columns:
-            raise ValueError(
-                "Only one of the parameters `nullify_missing_columns` (before `ignore_missing_columns`) and "
-                "`skip_missing_columns` can be set to True!"
-            )
+        self.missing_column_handling = missing_column_handling
         self.ignore_ambiguous_columns = ignore_ambiguous_columns
         self.mode = mode
 
+        if "ignore_missing_columns" in kwargs:
+            message = "Parameter `ignore_missing_columns` is deprecated, use `missing_column_handling` instead!"
+            if kwargs["ignore_missing_columns"] and missing_column_handling != "nullify":
+                raise ValueError(message)
+            else:
```

Comment:
If `ignore_missing_columns` is True and `missing_column_handling` is set to the default ("raise_error"), we probably should set the `column_handling` to `nullify` to ensure backwards compatibility for code that only sets the `ignore_missing_columns` parameter